### PR TITLE
Fix for invalid valuetype member state visibility

### PR DESCRIPTION
**Pull request by iguessthislldo**
Fix for #784 revealed after #723
In tao_idl, do not pass member visibility via the Bison stack indirectly.

### CODE REVIEW

**Comment by jwillemsen** [comment]
Can you extend one of our tests (for example TAO/tests/IDL_Test) with an example that failed before this fix? Would be good to have such a test at the TAO level

File: TAO/TAO_IDL/include/ast_field.h [hunk 1]
```diff
@@ -113,7 +113,12 @@ class TAO_IDL_FE_Export AST_Field : public virtual AST_Decl
 
   // Data Accessors.
   AST_Type *field_type (void) const;
-  Visibility visibility (void) const;
+
+  /// Get and Set Visibility
+  ///{
+  Visibility visibility () const;
```
Comment (mitza-oci) [inline TAO/TAO_IDL/include/ast_field.h]: Where is this method defined?

Comment (iguessthislldo) [reply TAO/TAO_IDL/include/ast_field.h]: [Here](https://github.com/DOCGroup/ACE_TAO/blob/3507f3c21ac55d6ffe1806a1b46fcbc7113bba55/TAO/TAO_IDL/ast/ast_field.cpp#L219), it's not new, I just moved the declaration to be with its counterpart.

**Comment by iguessthislldo** [comment]
> Can you extend one of our tests (for example TAO/tests/IDL_Test) with an example that failed before this fix? Would be good to have such a test at the TAO level

I'm not sure it's possible, unless I'm looking in the wrong place, the failures were in normal looking structs.

**Pull request merged**